### PR TITLE
Disable anonymous auth

### PR DESCRIPTION
**Pull request by rsevilla87**
### Description

- Disable anonymous authentication in elastic
- Enable TLS encrypted routes for Elastic, Thanos and Grafana

cc: @morenod @mohit-sheth @dry923

### CODE REVIEW

**Review by mkarg75 — APPROVED** [review]
LGTM!

File: charts/stacks/observability/values.yaml [hunk 1]
```diff
@@ -67,11 +56,6 @@ elastic:
         count: 3
         config:
           node.roles: ["data_warm", "ingest", "ml", "transform"]
-          xpack.security.authc:
```
Comment (Harshith-umesh) [inline charts/stacks/observability/values.yaml:70]: @rsevilla87 you also might need to delete lines 99-103 for this file

Comment (rsevilla87) [reply charts/stacks/observability/values.yaml:70]: good catch, fixed!

**Review by Harshith-umesh — APPROVED** [review]
LGTM! thanks raul for reporting and fixing this bug!

**Pull request merged**